Userland/Libraries/LibGfx/ImageFormats/ILBMLoader.cpp

```diff
@@ -225,12 +210,15 @@ static ErrorOr<void> decode_body_chunk(Chunk body_chunk, ILBMLoadingContext& con
 {
     dbgln_if(ILBM_DEBUG, "decode_body_chunk {}", body_chunk.data.size());
 
-    if (context.bm_header.compression == Cmp_ByteRun) {
+    if (context.BMHDHeader.compression == CompressionType::Cmp_ByteRun) {
         auto data = body_chunk.data;
         // these are the uncompressed interleaved bitmap planes
         auto plane_data = TRY(uncompress_byte_run(data, context));
         // that we need to convert to chunky pixel data
-        context.pixel_data = TRY(planar_to_chunky(plane_data, context));
+        auto pixel_data = TRY(planar_to_chunky(plane_data, context));
+
+        auto bitmap = TRY(chunky_to_bitmap(context, pixel_data));
+        context.bitmap = bitmap;
```

Comment:
No need for these two steps
```cpp
context.bitmap = TRY(chunky_to_bitmap(context, pixel_data));
```

Reply:
If you think I can directly write to the bitmap, I don't think so: I really need to de-interleave the bitplanes first, then only I can write to the bitmap. So I can put both steps into the same method, but these steps can't be put into a single step.

Reply:
I was speaking of the two-step assignment, the temporary `bitmap` variable is useless.

```cpp
auto bitmap = TRY(chunky_to_bitmap(context, pixel_data));
context.bitmap = bitmap;
```
is just
```cpp
context.bitmap = TRY(chunky_to_bitmap(context, pixel_data));
```
but with extra steps